langchain/src/base_language/index.ts

```diff
@@ -8,6 +8,7 @@ export {
   type BaseLanguageModelInput,
   type BaseLanguageModelInterface,
   BaseLanguageModel,
+  type FunctionDefinition,
```

Comment:
Instead of this, just import it from core inside `langchain/src/experimental/chat_models/ollama_functions.ts`
```suggestion
```